x/staking/types/params_legacy.go

```diff
@@ -0,0 +1,33 @@
+package types
+
+import paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
+
+var (
+	KeyUnbondingTime     = []byte("UnbondingTime")
+	KeyMaxValidators     = []byte("MaxValidators")
+	KeyMaxEntries        = []byte("MaxEntries")
+	KeyBondDenom         = []byte("BondDenom")
+	KeyHistoricalEntries = []byte("HistoricalEntries")
+	KeyMinCommissionRate = []byte("MinCommissionRate")
+)
+
+var _ paramtypes.ParamSet = (*Params)(nil)
+
+// ParamTable for staking module
+// NOTE: Deprecated.
```

Comment:
to follow godocs, we should add a paragraph starting with `Deprecated`. It's useful to also add what the replacement is